scanamo/src/main/scala/org/scanamo/DynamoFormat.scala

```diff
@@ -530,7 +543,7 @@ object DynamoFormat extends LowPriorityFormats {
     * }}}
     */
   implicit def mapFormat[V](implicit f: DynamoFormat[V]): DynamoFormat[Map[String, V]] =
-    xmap[Map[String, V], DynamoObject](_.toMap[V], m => DynamoObject(m.toSeq: _*))(javaMapFormat)
+    DynamoFormat.xmap[Map[String, V], DynamoObject](_.toMap[V], m => DynamoObject(m.toSeq: _*))(javaMapFormat)
```

Comment:
These can be reverted now.

Reply:
Done in bf3175964a839fa7565cd9d3c978c071809bcbc5